### ISSUE TITLE
Rum still gives normal drunkness with talent "Drunken Sailor"

### ISSUE THREAD

**Issue by Abkim27** [Regalis11/Barotrauma#12916]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When drinking rum by RMB it still gives normal drunkness, even though character has talent "Drunken Sailor". I had mods, but they had nothing to deal with this talent.

### Reproduction steps

1.Have a talent "Drunken Sailor"
2.Drink rum by RMB
3.Watch how you start to puke and lose health

### Bug prevalence

Everytime

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Linux

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/7ea66e491a9bcfc9e5dc5dfffdf33071f326256d

**Comment by ShotgunCrocodile** [comment]
Ran into this as well and was searching around. It seems like currently drunken sailor does nothing except allow you to craft rum at least in multiplayer where I've tried it. Drinking with right click, dragging via the health menu, clicking the use button above the inventory slot, and clicking the inventory number all result in normal drunken status and all the negative effects. I only found issues where it was marked as fixed and closed, what is the actual status currently?

**Comment by NilanthAnimosus** [comment]
This has been fixed but it has not been closed (the ticket is still open?). The current fix is not included in the current stable version or current unstable at this moment, some issues were found in testing and only after their resolved, and tested properly will the fix be included into an unstable/next version.

**Comment by Abkim27** [comment]
Oh, so the fix had been made, but it doesn't work properly, so they are still trying to fix? The talent didn't work properly in previous stable where i first found it, but occasianally you would sometimes get the no debuff version of drunkness and after that the normal drunkness is removed and you would gain more of no debuff version, that would happen after you drink ethanol via health menu? Also, when default drunkness is cured you would get medical skill, pretty funny that you can gain medical by drinking.

**Comment by NilanthAnimosus** [comment]
Yeah generally something is fixed then tested by quality assurance, if it creates new issues or the fix doesn't work fully more work is done on it, often a lot of these comments may end up where it can't actually be seen (IE. the private repo). The issue we found is it did not 1 to 1 convert the affliction properly with the new method added to convert the normal version of drunk with the reduced debuff version the talent should give. So 100%+ drunkness was only around 20-40%.

We'll often fix an issue, find a new issue. Release it once we think its good and end up missing another bug at times, but thats just how it goes.

Medical is gained with all medical items really, not sure if theres a way to regulate what specific afflictions give, or to exclude certain afflictions. As for the method of treatment, they should all be the same amount given.
